x-pack/plugins/data_usage/common/index.ts

```diff
@@ -12,6 +12,11 @@ export const PLUGIN_NAME = i18n.translate('xpack.dataUsage.name', {
   defaultMessage: 'Data Usage',
 });
 
+export const PAGING_PARAMS = Object.freeze({
+  default: 50,
+  all: 10000,
```

Comment:
is `all` being used?

Reply:
not at all. I'll remove it

Reply:
removed it 1e9f2ee3b6a82fc7dcdad2532608f4591ab61406